commitlint/fpHelpers.ts

```diff
@@ -1,3 +1,5 @@
+/* eslint @typescript-eslint/no-explicit-any: "off" */
```

Comment:
@webwarrior-ws why? unacceptable, let's fix it

Reply:
`public static IsNullOrUndefined(variable: any)`
`public static IsInstanceOf(variable: any, type: any)`
How do you propose to fix it? Are these methods not expecting literally `any` value?

Reply:
ah shit, didn't realise about this; can the no-explicit-any:off be applied per func instead of per file?

Reply:
> ah shit, didn't realise about this; can the no-explicit-any:off be applied per func instead of per file?

It can be applied for the next line.

Reply:
Then let's do that

Reply:
Already switched to using `eslint-disable-next-line` in `fpHelpers.ts` and `helpers.ts`.
For `commitlint.config.ts` I left original `/* eslint @typescript-eslint/no-explicit-any: "off" */` annotation because there are many places that use `any`.